### DISCUSSION TITLE
Problem using sdl2-image and libpng

### DISCUSSION THREAD

**Discussion by TrueFinch** [microsoft/vcpkg#28840] [Q&A] [answered]
I am tring to use sdl2-image package to create simple game engine.
But I got an error on calling IMG_Load: Failed loading libpng16.dll : module not found.
Figured out that cmake does not copy libpng16.dll and zlib1.dll to output directory (sdl2d.dll and sdl2_imaged.dll was copied without problem). I don't understand why. If i will copy libpng16.dll and zlib1.dll by hand then everithing works fine
Github repo: https://github.com/TrueFinch/picuma_course_engine

**Comment by dg0yt** [comment, marked as answer]
Did you try to either update vcpkg to the latest `HEAD`, or check existing issues?
#28073 was fixed one month ago.

**Reply by TrueFinch** [reply]
I am on 2022.11.14. Will try update right now

**Reply by TrueFinch** [reply]
Yes, everything is fine on version 2022-12-14. Thank you very much!
